nimterop.nimble

```diff
@@ -59,20 +59,24 @@ when (NimMajor, NimMinor, NimPatch) >= (0, 19, 9):
     fmt"{currentSourcePath}".parentDir.parentDir.parentDir
 
 proc runNimDoc() =
-  execCmd &"nim doc -o:{htmldocsDir} --project --index:on nimterop/all.nim"
-  execCmd &"nim buildIndex -o:{htmldocsDir}/theindex.html {htmldocsDir}"
-  when declared(getNimRootDir):
-    #[
-    this enables doc search, works at least locally with:
-    cd {htmldocsDir} && python -m SimpleHTTPServer 9009
-    ]#
-    execCmd &"nim js -o:{htmldocsDir}/dochack.js {getNimRootDir()}/tools/dochack/dochack.nim"
+  when not defined(windows): # pending https://github.com/nim-lang/Nim/pull/10629
```

Comment:
I generate docs on Windows usually, best not to disable altogether.

Reply:
made it work, using `patchFile`